supplementary_style_guide/style_guidelines/lead-in-sentences.adoc

```diff
@@ -0,0 +1,11 @@
+[[lead-in-sentences]]
+= Lead-in sentences
```

Comment:
@joaedwar We actually already have a placeholder spot for lead-in sentences. Would you mind copying the below content into that file under "== Lead-in sentences for `Prerequisites` and `Procedure` sections"? 

Here: https://raw.githubusercontent.com/redhat-documentation/supplementary-style-guide/master/supplementary_style_guide/style_guidelines/general-formatting.adoc